src/Symfony/Bundle/WebProfilerBundle/Tests/DependencyInjection/ConfigurationTest.php

```diff
@@ -19,6 +19,7 @@ class ConfigurationTest extends TestCase
 {
     /**
      * @dataProvider getDebugModes
+     * @group legacy
      */
     public function testConfigTree($options, $results)
```

Comment:
We need to keep this test as legacy indeed, but the test should be renamed (`testConfigTreeWithDeprecatedPosition`?) and have an `@expectedDeprecation`, then add the very same test (not legacy, named as this one currently) but without defining the `position` option.

Reply:
To keep only one data provider, I suggest removing the `position` option and re-add it in the legacy test (`$options['position'] = ...`)

Reply:
I like your idea, so I implemented it. Thanks.